### DISCUSSION TITLE
Automations

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#9891] [Help] [answered]
Hello everyone.
I have a problem that I can't run two or more automations at the same time.
For example I have an automation that will trigger after the creation of a new row, the purpose of this automation is to create another row after a delay of 1h45, in this interval of 1h45 I cannot work with other automations because the first one is still blocked in the 1:45 delay.
Is there a solution to this problem ?

**Comment by melohagan** [comment, marked as answer]
Hey @rafikApp 

You may be able to use the query rows to accomplish this.

So something roughly like so:

**Automation 1**
Update a boolean column on the created row to indicate that it needs updated later on.

**Automation 2**
 - Cron trigger (every few minutes)
 - Query Rows 
     - filter where boolean status column is true
     - filter where last updated was greater than 1h 45m ago (you'll probably need to do a JavaScript binding calculation here)
 - Create row action (with automation [looping](https://docs.budibase.com/docs/looping) for each row from the Query rows)
 - Update row (using looping as well for each entry in query rows) - set the boolean status column to false

Best of luck!
